pkg/cli/admin/release/release.go

```diff
@@ -2,11 +2,11 @@ package release
 
 import (
 	"github.com/spf13/cobra"
-
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	kcmdutil "k8s.io/kubectl/pkg/cmd/util"
 	"k8s.io/kubectl/pkg/util/templates"
 )
+//TODO: add --arch validation here so info, extract, and mirror can use all 4 arches?
 
```

Comment:
I think the `--arch` logic and validation belong here so mirror/info/extract can inherit this argument.. need to read up more on cobra and figure out the best way to do this
cc: @yselkowitz

Reply:
hmm..these are all individual commands so each of them need this option and logic in their command specific file i think

Reply:
but yes the validation function can go here